pilot/pkg/security/authz/model/model.go

```diff
@@ -265,3 +270,23 @@ func (m *Model) Generate(service *ServiceMetadata, forTCPFilter bool) *envoy_rba
 	}
 	return policy
 }
+
+// getPrincipalsIncludingAliases returns list of users encoded in spiffe format from the local trust domain
+// and its aliases.
+// For example, if the local trust domain is "cluster.local", and its aliases are "td1", "td2".
```

Comment:
There is nothing mention local trust domain in the function interface. Can you elaborate more?

Reply:
Comment mentions `local trust domain`, but there is no such thing in the function args nor body. It's good to document what is the `local trust domain` means in this context (i.e how its value is determined).

Reply:
I thought this was pretty thorough 
```
// getPrincipalsIncludingAliases returns a list of users encoded in SPIFFE format with the local
// trust domain and its aliases.
// For example, if the local trust domain is "cluster.local", and its aliases are "td1", "td2".
// For a user "bar" in namespace "foo", getPrincipalsIncludingAliases returns
// cluster.local/ns/foo/sa/bar, td1/ns/foo/sa/bar, td2/ns/foo/sa/bar
// The trust domain corresponds to the trust root of a system.
// Refer to
// [SPIFFE-ID](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain)
// In Istio, an identity is presented in the SPIFFE format, which is:
// spiffe://<trust-domain>/ns/<some-namespace>/sa/<some-service-account>
```
It mentions what a local trust domain is and how it is used in Istio identity/principal.